src/Config/Magento/Shared/Reader.php

```diff
@@ -46,6 +46,12 @@ public function read(): array
             return [];
         }
 
-        return require $configPath;
+        $content = require $configPath;
+
+        if (is_array($content)) {
+            return $content;
+        }
+
+        return [];
```

Comment:
I would use a ternary for this return statement, but this is still valid

Reply:
+1 ternary will look better
`return is_array($content) ? $content : [];`

Reply:
Changed